views/partials/footer.pug

```diff
@@ -0,0 +1,23 @@
+#footer
+
+    
+  //- CSS Style sheet for footer
+  style.
+    div {
+      text-align: center;
+    }
+
+  div © 2018 Kurt Lewis. Maintained by Kurt Lewis and Sean Kane.
+  div Code licensed under
+
+    //- Link to MIT License, might need to be updated?
+    a.navbar(href="http://opensource.org/licenses/mit-license.html") MIT License
```

Comment:
Something else I missed on this - pug removed the space between 'Code licensed under' and 'MIT License'. Idk how to fix - might take a little bit of playing with pug, but it could be as simple as adding a space after 'under'